### PR TITLE
calcite "Is null" filter

### PR DESCRIPTION
**Pull request by lgadde**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by mcjyang — APPROVED** [review]

File: service/src/test/scala/com/yahoo/maha/service/calcite/DefaultMahaCalciteSqlParserTest.scala [hunk 1]
```diff
@@ -425,4 +425,23 @@ class DefaultMahaCalciteSqlParserTest extends BaseMahaServiceTest with Matchers
...
+
+
+    val mahaSqlNode: MahaSqlNode = defaultMahaCalciteSqlParser.parse(sql, StudentSchema, "er")
+    assert(mahaSqlNode.isInstanceOf[SelectSqlNode])
+    val request = mahaSqlNode.asInstanceOf[SelectSqlNode].reportingRequest
+    assert(request.requestType === SyncRequest)
+    assert(request.filterExpressions.size > 0)
+
+
+    println("request " +  request.filterExpressions.toString)
```
Comment (pranavbhole) [inline service/src/test/scala/com/yahoo/maha/service/calcite/DefaultMahaCalciteSqlParserTest.scala:443]: remove println

**Review by pranavbhole — CHANGES_REQUESTED** [review]
remove println and empty lines in test, rest looks good

**Comment by pranavbhole** [comment]
and need rebase as well

File: service/src/test/scala/com/yahoo/maha/service/calcite/DefaultMahaCalciteSqlParserTest.scala (hunk 1 above)
Comment (lgadde) [reply service/src/test/scala/com/yahoo/maha/service/calcite/DefaultMahaCalciteSqlParserTest.scala]: done

**Comment by lgadde** [comment]
> and need rebase as well

**Review by pranavbhole — APPROVED** [review]

**Review by mcjyang — APPROVED** [review]

**Pull request merged**